lib/gtfs/components/GtfsFilter.js

```diff
@@ -64,50 +63,48 @@ export default class GtfsFilter extends Component {
             {allFeeds.length > 0
-             ? allFeeds.map((feed) => {
-               const isPublished = feed.publishedVersionId !== null
-               const disabled = loadedFeeds.findIndex(f => f.id === feed.id) === -1
-               return (
-                 <MenuItem
-                   key={feed.id}
-                   style={{width: '200px'}}
-                   eventKey={feed.id}
-                   alt={feed.shortName || feed.name}
-                   disabled={disabled}
-                   title={feed.shortName || feed.name}>
-                   <span>
-                     <Label
-                       bsStyle={disabled && isPublished
-                         ? 'warning'
-                         : disabled
-                         ? 'danger'
-                         : activeFeeds.indexOf(feed) === -1
-                         ? 'danger'
-                         : 'success'
-                       }
-                       className='pull-right'>
-                       {disabled && isPublished
-                         ? 'pending'
-                         : disabled
-                         ? 'unpublished'
-                         : activeFeeds.indexOf(feed) === -1
-                         ? 'off'
-                         : 'on'
-                       }
-                     </Label>
-                     <span>
-                       {feed.shortName || feed.name.length > 11
-                         ? feed.name.substr(0, 11) + '...'
-                         : feed.name
-                       }
-                     </span>
-                   </span>
-                 </MenuItem>
-               )
-             })
-            : <MenuItem disabled>
-             No feeds available
-            </MenuItem>
+              ? allFeeds.map((feed) => {
+                const isPublished = feed.publishedVersionId !== null
+                const disabled = loadedFeeds.findIndex(f => f.id === feed.id) === -1
+                return (
+                  <MenuItem
+                    key={feed.id}
+                    style={{width: '200px'}}
+                    eventKey={feed.id}
+                    alt={feed.shortName || feed.name}
+                    disabled={disabled}
+                    title={feed.shortName || feed.name}>
+                    <span>
+                      <Label
+                        bsStyle={disabled && isPublished
+                          ? 'warning'
+                          : disabled
+                            ? 'danger'
+                            : activeFeeds.indexOf(feed) === -1
+                              ? 'danger'
+                              : 'success'
+                        }
+                        className='pull-right'>
+                        {disabled && isPublished
+                          ? 'pending'
+                          : disabled
+                            ? 'unpublished'
+                            : activeFeeds.indexOf(feed) === -1
```

Comment:
make this activeFeeds.indexOf be a precalculated variable so it isn't calculated twice. Also would be a good place to use some().